index.html

```diff
@@ -6,12 +6,182 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" type="text/css" href="./styles/mainStyle.css">
   <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="stylesheet" type="text/css" href="./styles/mobileNav.css">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap" rel="stylesheet">
-  <title>My capstone project</title>
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <title>BuyGames!</title>
 </head>
 <body>
-  <h1>Hello</h1>
+  <header>
+    <nav class="navigator-menu">
+      <img class="navigator-menu__burger" src="./images/icons/burgerIcon.png"> 
+    </nav>
+
+    <!-- nav menu -->
+
+    <div class="hidden-menu">
+      <ul class="hidden-menu__ul">
+        <li class="hidden-menu__li x-container"><img class="hidden-menu__x-element" src="./images/xIcon.png"></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option" href="./about.html">About</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">Buy</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">Sell</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">Sponsor</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">News</a></li>
+        <li class="hidden-menu__li"><button type="submit" class="hidden-menu__button">Campain</button></li>
+      </ul>
+    </div>
+
+    <!-- Second nav bar -->
+
+    <div class="nav-bar-container">
+      <div class="nav-bar">
+        <div class="nav-bar__icons"><img src="./images/facebook.png"></div>
+        <div class="nav-bar__icons"><img src="./images/twitter.png"></div>
+        <div class="nav-bar__language"><p class="nav-bar__english">English</p></div>
+      </div>
```

Comment:
- [ ] All links in the header section should be inside a `nav` tag. This is the ideal way to semantically structure your content in HTML which is the best practice followed 👍